### ISSUE TITLE
Only list certain file extensions

### ISSUE THREAD

**Issue by sckott** [rufuspollock/s3-bucket-listing#32]
I have [a static site](http://ropensci-windows.s3-website-us-west-2.amazonaws.com/) on S3, and want to list only the `.zip` files, and not my `index.html` and `404.html` files. Is there currently a way to do that? Or maybe there's an alternative way, like putting all `.zip` files in a folder, then setting the path for listing files to `root/folder/`

**Comment by rufuspollock** [comment]
I do not think this is that hard to do. Depends on s3 api and whether you could hack this locally. Wonder if related to #2
